americangut/util.py

```diff
@@ -178,15 +261,29 @@ def fetch_metadata_xml(accession):
     metadata = {}
     for node in attributes.iterfind('SAMPLE_ATTRIBUTE'):
         tag, value = node.getchildren()
-        metadata[tag.text.strip('" ').upper()] = value.text.strip('" ')
+        if value.text is None:
+            metadata[tag.text.strip('" ').upper()] = 'no_data'
+        else:
+            metadata[tag.text.strip('" ').upper()] = value.text.strip('" ')
+
+    description = sample.find('DESCRIPTION')
+    metadata['Description'] = description.text.strip('" ')
+
     return metadata
 
 
-def fetch_study(accession, metadata_path, fasta_path):
+def fetch_study(accession):
     """Fetch and dump a full study
 
```

Comment:
this is old code, so i agree that refreshes on it are needed but not the immediate focus.